R/stat-smooth.r

```diff
@@ -1,5 +1,6 @@
-#' @param method Smoothing method (function) to use, eg. `lm`, `glm`,
-#'   `gam`, `loess`, `MASS::rlm`.
+#' @param method Smoothing method (function) to use, accepts either a character vector,
+#'   e.g. `"auto"`, `"lm"`, `"glm"`, `"gam"`, `"loess"` or a function, e.g.
+#'   `MASS::rlm` or `mgcv::gam`, `lm`, or `loess`.
```

Comment:
Do you mean to have `lm` and `loess` in both places? If yes, is that because they are base functions? If so, I'd match them with `MASS::rlm` by prefixing with `base::`